index.html

```diff
@@ -115,6 +115,29 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <div class="card">
+          <p class="name">Ahammed Imthias</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="https://twitter.com/LeviAckerman__l">Levi_ackerman_l</a>
+          </p>
+          <p class="about">about you</p>
```

Comment:
Write something about your self here...
```suggestion
          <p class="about">about you</p>
```